pages/staking.staking.tsx

```diff
@@ -160,6 +191,28 @@ export default function Staking() {
                   hasDiscountProgram={true}
                 />
               </Grid>
+              <Grid
+                item
+                xs={12}
+                lg={6}
+                sx={{ display: { xs: isStakeAAVE ? 'none' : 'block', lg: 'block' } }}
+              >
+                <StakingPanel
+                  stakeTitle="GHO"
+                  stakedToken="GHO"
+                  maxSlash="1" // 100%
+                  icon="gho"
+                  stakeData={stkGho}
+                  stakeUserData={stkGhoUserData}
+                  ethPriceUsd={stakeGeneralResult?.ethPriceUsd}
+                  onStakeAction={() => openStake('gho', 'gho')}
```

Comment:
Should be 
```
openStake('gho', 'GHO')
```
The second param is what shows in the action buttons for the text, so all caps will be consistent with the others. Same thing with the `openUnstake` method